### DISCUSSION TITLE
無法開啟ImageTrans.exe，點擊沒有反應

### DISCUSSION THREAD

**Discussion by Jasonnothing** [xulihang/ImageTrans-docs#513] [Q&A]
解壓縮ImageTrans-crossplatform後，點擊ImageTrans.exe沒有反應
<img width="666" alt="未命名" src="https://github.com/xulihang/ImageTrans-docs/assets/43427609/0ed0096e-8244-40eb-b6a0-bf8504057a5f">

**Comment by xulihang** [comment]
照邮件和文档里说的，下载onedrive里的完整版，可以直接运行。跨平台版需要自己安装额外的运行环境。

**Reply by Jasonnothing** [reply]
已解決問題，使用onedrive的檔案即可開啟，感謝作者
<img width="1005" alt="未命名" src="https://github.com/xulihang/ImageTrans-docs/assets/43427609/97532fe0-fd33-4e86-92e2-707fa972077e">
